File: BlazorWithSharpRedux/wwwroot/devTools.js
const withDevTools = (typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION__);
Blazor.registerFunction('DevToools.AreAvailable', function () {
    return withDevTools !== null;
});
Blazor.registerFunction('DevToools.Init', function (state) {
    console.log('Init for : ' + withDevTools);
    window.devTools = window.__REDUX_DEVTOOLS_EXTENSION__.connect();
    return devTools.init(state);
});
Blazor.registerFunction('DevToools.Send', function (action, state) {
    return devTools.send(action, state);
});

File: BlazorWithSharpRedux/Services/DevTools.cs
using Microsoft.AspNetCore.Blazor.Browser.Interop;

namespace BlazorWithSharpRedux.Services
{
    public static class DevTools
    {
        public static bool AreAvailable()
        {
            return RegisteredFunction.Invoke<bool>("DevToools.AreAvailable");
        }
        public static void Init(object state)
        {
            RegisteredFunction.Invoke<object>("DevToools.Init", state);
        }
        public static void Send(object action, object state)
        {
            RegisteredFunction.Invoke<object>("DevToools.Send", action, state);
        }
    }
}
